core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -251,13 +249,12 @@ coreos:
       command: start
       content: |
         [Unit]
-        Wants={{.Etcd.SystemdUnitName}}
+        Requires={{.Etcd.SystemdUnitName}}
```

Comment:
@redbaron Argh, maybe I had forgotten about that multiple times until now!
Thanks for the confirmation 👍 
LGTM for now.

In the future, perhaps we'd better transform all the oneshot services into simple services with all the useful commands in "ExecStartPre"s, while having `ExecStart=/bin/true`?